apps/web-giddh/src/app/financial-reports/components/profit-loss/profit-loss.component.ts

```
@@ -59,133 +65,163 @@ export class ProfitLossComponent implements OnInit, AfterViewInit, OnDestroy {
-                            c.isCreated = true;
-                            c.isIncludedInSearch = true;
-                        });
-                    });
-                }
+            this.cogsData = cogsGrp;
+        }
 
-                if (data?.incomeStatement?.grossProfit?.type === "DEBIT" && data.incomeStatement.grossProfit.amount) {
-                    data.incomeStatement.grossProfit.amount = "-" + data.incomeStatement.grossProfit.amount;
-                }
+        if (data && data.expArr) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)